barebones/core_portme.mak

```diff
@@ -1,87 +0,0 @@
-# Copyright 2018 Embedded Microprocessor Benchmark Consortium (EEMBC)
```

Comment:
Why delete this?

Reply:
Just for consistency across testsuites, this didn't expose any special compiling requirements other than -O0 and is verbose for what we need.

Reply:
Leave it in just as a reference, we won’t use it though

Reply:
Ok I'll readd it